metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/configuration/metron-env.xml

```diff
@@ -37,6 +37,12 @@
         <display-name>Metron apps indexed HDFS dir</display-name>
     </property>
     <property>
```

Comment:
Actually you can control the file name in the index config so separate index configs solves that